common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,8 +1,8 @@
-@(file: String, path: String = "", classes: Seq[String] = List(), title: Option[String] = None)
+@(file: String, path: String = "", classes: Seq[String] = List(), title: Option[String] = None, isPresentation: Boolean = false)
```

Comment:
@SiAdcock reviewing myself here. What if instead of `isPresentation: Boolean` I made it `description: Option[String] = None`? 

If there's a description it uses it as alt text, if there's nothing it puts `aria-hidden` as it is now

I don't think any of our svgs contain alt text inline (probably shouldn't) so I don't think there's any point in making screen readers try to digest them unless they are explicitly described?

Reply:
Interesting. Can we make it `title`, and use it to add a [`<title>` element](https://css-tricks.com/accessible-svgs/#article-header-id-9) to the SVG?

I'm somewhat uncomfortable adding `aria-hidden="true"` to every existing inline SVG without assessing the impact. What would happen to SVGs used as links? Would screen readers not see the link at all?

I would rather exercise caution and keep the `isPresentation: Boolean = false` parameter, rather than tying `aria-hidden="true"` to the existence of a `title`.

Reply:
okay!